buildclient.cmd

```diff
@@ -0,0 +1 @@
+dotnet run --project Content.Client --configuration RELEASE
```

Comment:
What is this?

Reply:
FUCK forgot to delete it

Reply:
do I have to make another commit without it or can it just be removed on here?

Reply:
Make a new commit that deletes the two errant files.